src/components/categoriesCarousel/categoriesCarousel.css

```diff
@@ -0,0 +1,38 @@
+.categories-carousel {
+  padding: 20px 30px;
+  margin: 10px;
+}
+.categories-carousel__container {
+  padding: 0px 40px;
+  text-align: center;
+  font-size: 14px;
+  color: #717171;
+  cursor: pointer;
+  overflow-x: scroll;
+}
+.categories-carousel__container-list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  gap: 20px;
+}
+.categories-carousel__container-list-image {
+  height: 80px;
+  width: 90px;
+}
+@media screen and (max-width: 1024px) {
+  .categories-carousel {
+    padding: 0;
+  }
+  .categories-carousel__container {
+    padding: 0px;
+  }
+  .categories-carousel__container-list-image {
+    height: 60px;
+    width: 70px;
+  }
+}
+.categories-carousel__container::-webkit-scrollbar {
```

Comment:
this component and the filter are not scrolling in the responsive version, please remember that always test the responsive version :( 

I think this occurred because the scroll is none, maybe you should keep the scroll visible but add a style just to make it look better, like: https://www.w3schools.com/howto/howto_css_custom_scrollbar.asp
you can use this width property to decrease the scroll width for example. 





![scroll](https://user-images.githubusercontent.com/38358239/158456040-c1eb4d64-93e2-42f7-a889-50f6083b3b27.gif)

Reply:
really, I adjusted, the problem was that it had fixed padding and margin:

https://user-images.githubusercontent.com/42494406/159073019-73645f6d-a887-418c-a501-9d04ac10f38d.mov

Reply:
I saw that there is a bug in the header address text when the screen is zoomed out, I will see and fix it in another PR as I don't think it makes much sense to fix it in this PR 🤔

Reply:
here it's still not scrolling, did you push the changes? 🤔

Reply:
yes rai, that's weird, I'll look here